src/api/spec/lib/backend/api/search_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Backend::Api::Search, vcr: true do
+  describe '.projects' do
+    context 'with projects' do
+      let(:project_names) { 2.times.collect { |i| create(:project, name: "foo_#{i}") }.map(&:name) }
```

Comment:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#building-or-creating-multiple-records

Reply:
It's one of the suggestions from my [previous comment](https://github.com/openSUSE/open-build-service/pull/9783#discussion_r442133313). @vpereira told that it didn't work with `create_list` due to the backend. I wonder if it's because it should be `let!` instead of `let`.